docs/advanced/swap.md

```diff
@@ -226,3 +226,7 @@ If running ./cashout.sh returns nothing, you currently have no uncashed cheques.
 :::info
 Are you a Windows-user who is willing to help us? We are currently missing a simple cashout script for Windows. Please see the [issue](https://github.com/ethersphere/bee/issues/1092)
 :::
+
```

Comment:
"Merge branch 'master' into cashout"?

it would be nice to avoid such random merge commits, but it's fine with me if you guys don't mind.

Reply:
There was a conflict